x/feemarket/keeper/eip1559.go

```diff
@@ -16,19 +16,24 @@ import (
 func (k Keeper) CalculateBaseFee(ctx sdk.Context) *big.Int {
 	params := k.GetParams(ctx)
 
-	// Ignore the calculation if not enable
+	// Ignore the calculation if not enabled
 	if !params.IsBaseFeeEnabled(ctx.BlockHeight()) {
 		return nil
 	}
 
 	consParams := ctx.ConsensusParams()
 
-	// If the current block is the first EIP-1559 block, return the InitialBaseFee.
+	// If the current block is the first EIP-1559 block, return the base fee
+	// defined in the parameters (DefaultBaseFee if it hasn't been changed by
+	// governance).
 	if ctx.BlockHeight() == params.EnableHeight {
 		return params.BaseFee.BigInt()
 	}
 
 	// get the block gas used and the base fee values for the parent block.
+	// NOTE: this is not the parent's base fee but the current block's base fee,
+	// as it is retrieved from the transient store, which is committed to the
+	// persistent KVStore after EndBlock (ABCI Commit).
```

Comment:
cc @jolube

Reply:
Should the variable `parentBaseFee` be renamed in this case?